src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -8,14 +8,85 @@
 
 namespace System
 {
-   public partial class AppDomain
-   {
-       private AppDomain() {}
-       public static AppDomain CurrentDomain { get { return default(AppDomain); } }
-       public string BaseDirectory { get { return default(string); } }
-       public string RelativeSearchPath { get { return default(string); } }
-       public event System.UnhandledExceptionEventHandler UnhandledException { add { } remove { } } 
-   }
+    public partial class AppDomain : System.MarshalByRefObject
+    {
+        private AppDomain() {}
+        public static AppDomain CurrentDomain { get { return default(AppDomain); } }
+        public string BaseDirectory { get { return default(string); } }
+        public string RelativeSearchPath { get { return default(string); } }
+        public event System.UnhandledExceptionEventHandler UnhandledException { add { } remove { } } 
+        public string DynamicDirectory { get { throw null; } }
+        public string FriendlyName { get { throw null; } }
+        public int Id { get { throw null; } }
+        public bool IsFullyTrusted { get { throw null; } }
+        public bool IsHomogenous { get { throw null; } }
+        public static bool MonitoringIsEnabled { get { throw null; } set { } }
+        public long MonitoringSurvivedMemorySize { get { throw null; } }
+        public static long MonitoringSurvivedProcessMemorySize { get { throw null; } }
+        public long MonitoringTotalAllocatedMemorySize { get { throw null; } }
+        public System.TimeSpan MonitoringTotalProcessorTime { get { throw null; } }
+        public bool ShadowCopyFiles { get { throw null; } }
+        public event System.AssemblyLoadEventHandler AssemblyLoad { add { } remove { } }
+        public event System.ResolveEventHandler AssemblyResolve { add { } remove { } }
+        public event System.EventHandler DomainUnload { add { } remove { } }
+        public event System.EventHandler<System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs> FirstChanceException { add { } remove { } }
+        public event System.EventHandler ProcessExit { add { } remove { } }
+        public event System.ResolveEventHandler ResourceResolve { add { } remove { } }
+        public event System.ResolveEventHandler TypeResolve { add { } remove { } }
+        public void AppendPrivatePath(string path) { }
```

Comment:
Yes anything marked obsolete here https://github.com/dotnet/standard/blob/master/netstandard/ref/mscorlib.appdomain.cs#L38 should also be obsolete in this ref.